tests/test_import_queue.py

```diff
@@ -73,49 +70,18 @@ def mock_bulk_insert_application_records(mocker, mock_sqs_recieve_message):
         yield
 
 
-class TestSQSFunctions(object):
-    @pytest.mark.fundround("NSTFR2")
-    @pytest.mark.appcount(3)
-    def test_receive_messages(self, request, mock_sqs_recieve_message):
-        appcount = request.node.get_closest_marker("appcount").args[0]
-
-        # Call the function
-        messages = receive_messages(
-            UnitTestingConfig.AWS_PRIMARY_QUEUE_URL, max_number=2
-        )
-
-        # Assertions
-        assert len(messages) == appcount
-        assert messages[0]["MessageId"] == "1"
-
-    @pytest.mark.appcount(3)
-    def test_delete_messages(self, request, mock_sqs_delete_message):
-        appcount = request.node.get_closest_marker("appcount").args[0]
-
-        # Call the function
-        receipt_handles = [
-            "receipt_handle_" + str(count) for count in range(appcount)
-        ]
-        response = delete_messages(
-            UnitTestingConfig.AWS_PRIMARY_QUEUE_URL, receipt_handles
-        )
-
-        # Assertions
-        assert len(response["Successful"]) == appcount
-
-    @pytest.mark.fundround("NSTFR2")
-    @pytest.mark.appcount(3)
-    def test_import_application_queue(
-        self,
-        request,
-        mock_sqs_recieve_message,
-        mock_sqs_delete_message,
-        mock_bulk_insert_application_records,
-    ):
-        appcount = request.node.get_closest_marker("appcount").args[0]
+@pytest.mark.fundround("NSTFR2")
+@pytest.mark.appcount(3)
+def test_import_application_queue(
+    request,
+    mock_sqs_recieve_message,
+    mock_sqs_delete_message,
+    mock_bulk_insert_application_records,
+):
+    appcount = request.node.get_closest_marker("appcount").args[0]
 
-        # Call the function
-        response = import_applications_from_queue()
+    # Call the function
+    response = import_applications_from_queue()
 
-        # Assertions
-        assert len(response) == appcount
+    # Assertions
+    assert len(response) == appcount
```

Comment:
Not a fan of the comments since they're just duplicating what the code is saying

Reply:
Just realised this isn't part of your PR though, just indentation changes! So feel free to ignore